cassandra/cluster.py

```diff
@@ -772,24 +796,18 @@ def __init__(self,
 
         Any of the mutable Cluster attributes may be set as keyword arguments to the constructor.
         """
-        if contact_points is not _UNSET_ARG and load_balancing_policy is None:
-            log.warn('Cluster.__init__ called with contact_points specified, '
-                     'but no load_balancing_policy. In the next major '
-                     'version, this will raise an error; please specify a '
-                     'load balancing policy. '
-                     '(contact_points = {cp}, lbp = {lbp}'
-                     ''.format(cp=contact_points, lbp=load_balancing_policy))
-
         if contact_points is not None:
-            if contact_points is _UNSET_ARG:
-                contact_points = ['127.0.0.1']
+            if contact_points is _NOT_SET:
+                _contact_points_to_set = ['127.0.0.1']
+            else:
+                _contact_points_to_set = contact_points
```

Comment:
Do we need that variable? Can't we just use `self.contact_points` and `contact_points`  accordingly?

Reply:
Is this what you're proposing?
 
```python
        if contact_points is not None:
            if contact_points is _NOT_SET:
                self.contact_points = ['127.0.0.1']
            else:
                self.contact_points = contact_points

            if isinstance(self.contact_points, six.string_types):
                raise TypeError("contact_points should not be a string, it should be a sequence (e.g. list) of strings")

            if None in self.contact_points:
                raise ValueError("contact_points should not contain None (it can resolve to localhost)")
            self.contact_points = _contact_points_to_set
```

Yeah, that seems better.

Reply:
:+1: